llvm/test/Transforms/InstCombine/icmp-of-and-x.ll

```diff
@@ -238,9 +238,9 @@ define i1 @icmp_sle_negx_y_fail_maybe_zero(i8 %x, i8 %y) {
 
 define i1 @icmp_eq_x_invertable_y_todo(i8 %x, i1 %y) {
```

Comment:
I don't know what does the `TODO` mean here.

Reply:
It was todo before this patch.

Reply:
It would be good to rename the function and add some header comments :)

Reply:
Oh no, these are from before we had `isFreeToInvert...`
This is from a year-old series (where `isFreeToInvert...` was later on).